app/helpers/notes_helper.rb

```diff
@@ -22,7 +22,7 @@ def note_timestamp(note)
       ts << capture_haml do
         haml_tag :span do
           haml_concat '&middot;'
-          haml_concat '<i class="fa fa-edit" title="edited"></i> '
+          haml_concat icon('edit', title: "edited")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.